File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.cpp
class Solution {
public:
    int maxProduct(int n) {
        vector<int>arr;

        while(n>0){
            int k=n%10;
            arr.push_back(k);
            n=n/10;
        }
        sort(arr.begin(),arr.end());
        int p=arr.size();
        return arr[p-1]*arr[p-2];
    }
};

File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.py
class Solution(object):
    def maxProduct(self, n):
        """
        :type n: int
        :rtype: int
        """
        arr=[]
        while n>0:
            k=n%10
            arr.append(k)
            n=n/10
        
        arr.sort()

        return arr[-1] * arr[-2]